src/tx/applyTx/checkSpendCond.js

```diff
@@ -393,16 +391,21 @@ module.exports = async (state, tx, bridgeState, nodeConfig = {}) => {
     });
   });
 
-  const evmResult = await runTx(vm, {
-    nonce: nonceCounter,
-    gasLimit: GAS_LIMIT_HEX, // TODO: set gas Limit to (inputs - outputs) / gasPrice
-    to: sigHashBuf,
-    // NOPE: the plasma address is replaced with sighash, to prevent replay attacks
-    data: spendingInput.msgData,
-  });
+  let evmResult;
 
-  const logOuts = [];
+  try {
+    evmResult = await runTx(vm, {
+      nonce: nonceCounter,
+      gasLimit: GAS_LIMIT_HEX, // TODO: set gas Limit to (inputs - outputs) / gasPrice
+      to: sigHashBuf,
+      // NOPE: the plasma address is replaced with sighash, to prevent replay attacks
+      data: spendingInput.msgData,
+    });
+  } catch (err) {
+    console.log(err);
```

Comment:
i wasn't receiving any information about EVM errors without this try catch 🤷‍♂️

Reply:
shouldn't we rethrow here to preserve backward compatiblity?

Reply:
Good point 🙂